LICENSES/GPL-2.0-only.txt

```diff
@@ -51,7 +51,7 @@ In addition, mere aggregation of another work not based on the Program with the
 
     a) Accompany it with the complete corresponding machine-readable source code, which must be distributed under the terms of Sections 1 and 2 above on a medium customarily used for software interchange; or, 
     b) Accompany it with a written offer, valid for at least three years, to give any third party, for a charge no more than your cost of physically performing source distribution, a complete machine-readable copy of the corresponding source code, to be distributed under the terms of Sections 1 and 2 above on a medium customarily used for software interchange; or, 
-    c) Accompany it with the information you received as to the offer to distribute corresponding source code. (This alternative is allowed only for noncommercial distribution and only if you received the program in object code or executable form with such an offer, in accord with Subsection b above.) 
```

Comment:
Aside from the fact that modifying the license is a copyright violation (

> Everyone is permitted to copy and distribute verbatim copies
> of this license document, but changing it is not allowed.

) and that it could be considered a relicensing (which needs agreement from everyone which contributed code under said license, disclaimer: IANAL), "in accord with" is actually valid English: https://dictionary.cambridge.org/dictionary/english/in-accord-with